localization/french/culture/wc_culture_titles_l_french.yml

```
@@ -89,6 +88,15 @@
  grand_admiral_female_tirassian_emperor:0 "Grand Amiral"
  grand_admiralty_tirassian_empire:0 "Grande Amirauté"
 
+ ### Gnomes ###
+ # Republic
+ high_tinker_male_king:0 "Haut Bricoleur"
+ high_tinker_female_king:0 "Haute Bricoleuse"
```

```suggestion
 high_tinker_female_king:0 "Grande Bricoleuse"
```
La trad officielle fait bien ça...

---

Du coup je pense quand même que tu peux garder la même chose pour le tier empereur.